R/authenticate_sb.R

```diff
@@ -47,6 +47,7 @@ authenticate_sb = function(username, password){
 		stop('Unable to authenticate to SB. Check username and password')
 	}
 	
+	attributes(h) <- c(attributes(h), list(birthdate=Sys.time()))
```

Comment:
yeah, that was one way I could avoid breaking API for session pass through stuff.